README.md

```diff
@@ -1 +1,27 @@
-# cleaner-mobile-app
\ No newline at end of file
+# Cleaner Web App
+
+React front end for cleaner application
+
+## Available Scripts
+
```

Comment:
# Should tell the user they need node and npm installed.
# tell the user to run npm install